README.md

```diff
@@ -219,6 +219,20 @@ debug symbols included.
 Provide additional arguments and options to `make` by using
 `--makeopts`.
 
+#### Kernel configuration file options
+
+Three kernel configuration file options are supported by `skt`:
+
+* Provide a kernel configuration file directly: `--config CONFIG_FILE_PATH`
+* Use a minimal configuration for a very small kernel: `--cfgtype tinyconfig`
+* Build kernel configuration files for Red Hat kernels: `--cfgtype rh-configs`
+
+When `rh-configs` is used, `skt` checks to see the `ARCH` environment
```

Comment:
should this be `*if* the `ARCH` environment`? I'm not a native speaker, but find it hard to parse :)